_config/products.yml

```diff
@@ -462,11 +462,39 @@ jbt_core:
             #md5_url: .MD5
 
     luna:
+      4.2.3.Final:
+        update_site_url: http://download.jboss.org/jbosstools/updates/stable/luna/
+        marketplace_url: http://marketplace.eclipse.org/node/1617241
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+        #archived: true
+        release_date: 2015-04-01
+        supported_jbt_is_version: 4.2.0.Final
+        blog_announcement_url: /blog/2015-04-01-devstudio-8.1.0.GA-for-luna.html
```

Comment:
@mickaelistria I renamed the blog to /blog/2015-04-02-devstudio-8.1.0.GA-for-luna.html (April 2nd) because the JBDS release was delayed. Sorry for the confusion but I can fix it when will be pushing to production (when JBDS is ready).
See https://github.com/jbosstools/jbosstools-website/pull/372